Task 8

misostc opened the pull request.

misostc committed:
Task 8

misostc committed:
Added test

vandus reviewed (changes requested):
appka po kliku na detail hadze error:

 4702-4711/cz.muni.fi.pv256.movio2.uco_410434.alternative.debug E/StrictMode: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.
                                                                                                java.lang.Throwable: Explicit termination method 'close' not called
                                                                                                    at dalvik.system.CloseGuard.open(CloseGuard.java:223)
                                                                                                    at android.content.ContentResolver$CursorWrapperInner.<init>(ContentResolver.java:2925)
                                                                                                    at android.content.ContentResolver.query(ContentResolver.java:780)
                                                                                                    at android.content.ContentResolver.query(ContentResolver.java:704)
                                                                                                    at android.content.ContentResolver.query(ContentResolver.java:662)
                                                                                                    at cz.muni.fi.pv256.movio2.uco_410434.data.PersistedFilmManager.getFilms(PersistedFilmManager.java:42)
                                                                                                    at cz.muni.fi.pv256.movio2.uco_410434.ui.PersistedFilmListFragment$LoadFilmsAsyncTask.doInBackground(PersistedFilmListFragment.java:35)
                                                                                                    at cz.muni.fi.pv256.movio2.uco_410434.ui.PersistedFilmListFragment$LoadFilmsAsyncTask.doInBackground(PersistedFilmListFragment.java:29)
                                                                                                    at android.os.AsyncTask$2.call(AsyncTask.java:333)
                                                                                                    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
                                                                                                    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
                                                                                                    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
                                                                                                    at java.lang.Thread.run(Thread.java:764)

misostc committed:
Task 8 fixes

Fixed:
- code style issues from task 7
- closing of db cursor
- fragment switch issue in film list
- performance problem with picasso resize

misostc commented:
Ahoj nepodarilo sa mi to zreplikovat co si pisala, ale myslim, ze co som pridal do opravy by mohlo pomoct, tiez som opravil veci co si komentovala z predchadzajuceho tasku

vandus reviewed (changes requested):
No neviem, mas tam nieco dost dost zle.

Stale mam velmi velke tie obrazky

FATAL EXCEPTION: main
                                                                                                    Process: cz.muni.fi.pv256.movio2.uco_410434.alternative.debug, PID: 7754
                                                                                                    java.lang.RuntimeException: Canvas: trying to draw too large(122463664bytes) bitmap.
                                                                                                        at android.view.DisplayListCanvas.throwIfCannotDraw(DisplayListCanvas.java:229)
                                                                                                        at android.view.RecordingCanvas.drawBitmap(RecordingCanvas.java:97)
                                                                                                        at android.graphics.drawable.BitmapDrawable.draw(BitmapDrawable.java:529)
                                                                                                        at com.squareup.picasso.PicassoDrawable.draw(PicassoDrawable.java:102)
                                                                                                        at android.widget.ImageView.onDraw(ImageView.java:1367)
                                                                                                        at android.view.View.draw(View.java:19123)
                                                                                                        at android.view.View.updateDisplayListIfDirty(View.java:18073)
                                                                                                        at android.view.View.draw(View.java:18851)
                                                                                                        at android.view.ViewGroup.drawChild(ViewGroup.java:4214)
                                                                                                        at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4000)


Skius sa pozriet na tento projekt , on to ma spravene dobre. Aj performance aj vsetko. 
https://github.com/Lukas-Razz/PV256-Project
